lib/widgets/sidelayout.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+
+class SideLayout extends StatefulWidget {
+  @override
+  SideLayoutState createState() => new SideLayoutState();
+}
+
+class SideLayoutState extends State<SideLayout> {
+  Map<String, bool> values = {
+    'chest': false,
+    'back': false,
+    'arms': false,
+    'shoulders': false,
+    'legs': false,
+    'calves': false,
+    'abs': false,
+    'neck': false,
+    'hips': false,
+    'tummy': false
+  };
+
+  @override
+  Widget build(BuildContext context) {
+    return Expanded(
+      child: Container(
+          padding: EdgeInsets.symmetric(vertical: 20, horizontal: 20),
+          color: Color(0x00000000),
+          child: Column(
+            children: [
+              Row(
```

Comment:
This is giving me a overflow error when the page is shrunk horizontally
![Capture](https://user-images.githubusercontent.com/31574865/144498226-8cabd7d1-ee6d-4556-9f2b-a88c40789893.PNG)
.

Reply:
Added ticket for this